pipelines/optimus/Optimus.wdl

```diff
@@ -21,8 +21,11 @@ workflow Optimus {
   meta {
     description: "The optimus 3' pipeline processes 10x genomics sequencing data based on the v2 chemistry. It corrects cell barcodes and UMIs, aligns reads, marks duplicates, and returns data as alignments in BAM format and as counts in sparse matrix exchange format."
   }
-  
+
   input {
+    # Mode for counting either "sc_rna" or "sn_rna"
+    String counting_mode = "sc_rna"
+
     # version of this pipeline
     String version = "optimus_v2.0.0"
```

Comment:
The version can be overridden if it is defined within the input block, I think it should be outside the input block right after the workflow definition

Reply:
@jessicaway @pshapiro4broad can you please expand on this?

Reply:
Yes, exactly. Phil described that much better

Reply:
Fixed, btw this was not changed in this PR, but its good that we fixed it here